packages/gasket-cli/src/help.js

```diff
@@ -0,0 +1,22 @@
+const Help = require('@oclif/plugin-help').default;
+
+module.exports = class GasketHelp extends Help {
+  showHelp(args) {
+    console.log(`
```

Comment:
@3rd-Eden This logo already exists, but [only in the `create` lifecycle](https://github.com/godaddy/gasket/blob/master/packages/gasket-cli/src/scaffold/actions/print-report.js#L70-L75). Was the purpose of this work to have the logo run for *every* command? If so, shouldn't this PR also remove that bit of code?

Reply:
Probably don't want to run it on **every** command, e.g. it has no point for `gasket start` which run in production environments, but it wouldn't hurt to display it during other development commands.

Reply:
Given that a text logo already exists elsewhere, we should probably use a single source of logo truth. So then two questions:

1. Which version of the logo should we use? (I prefer the new one, but I may be biased 😉 )
2. What other command(s) should display this? I only see `create` under the `commands/` folder, so I must be missing something if there is also a `start` command...

Reply:
I think it would be overwhelming to have the logo output on every command. `create` feels like the appropriate time for it.